src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -71,7 +71,10 @@ internal void InitializeCurl()
                 SetVerb();
                 SetDecompressionOptions();
                 SetProxyOptions(_requestMessage.RequestUri);
```

Comment:
Yes, but at the moment,  this fix is intended to be the minimal change needed to unblock wcf tests on linux.  For proxy credentials, there is an immediate work around since libcurl can take proxy creds from Environment variable.